### PR TITLE
Fix #1412: Review liquibase scripts

### LINKED ISSUE #1412: Review liquibase scripts

### PR DESCRIPTION
**Pull request by jandusil**
- Review sequences

### CODE REVIEW

**Review by romanstrobl — CHANGES_REQUESTED** [review]
This will probably end up with many errors like this:

```
ERROR: value "9999999999999999999999999999" is out of range for type bigint
```

**Review by romanstrobl — CHANGES_REQUESTED** [review]
The `minValue` setting seems to be inconsistent.

File: docs/db/changelog/changesets/powerauth-nextstep/1.4.x/20230324-init-db.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
             </not>
         </preConditions>
         <comment>Create a new sequence ns_operation_afs_seq</comment>
-        <createSequence sequenceName="ns_operation_afs_seq" cacheSize="20" />
+        <createSequence sequenceName="ns_operation_afs_seq" startValue="1" incrementBy="1" cacheSize="20" minValue="1" />
```
Comment (romanstrobl) [inline docs/db/changelog/changesets/powerauth-nextstep/1.4.x/20230324-init-db.xml:13]: We set `minValue` here however not in other sections. We should probably always se this.

Comment (romanstrobl) [reply docs/db/changelog/changesets/powerauth-nextstep/1.4.x/20230324-init-db.xml:13]: Ah, I see there is already the `startValue`, so this setting seems rather redundant. Anyway it is not consistent.

Comment (jandusil) [reply docs/db/changelog/changesets/powerauth-nextstep/1.4.x/20230324-init-db.xml]: Apologies, this was forgotten by mistake

**Review by zcgandcomp — APPROVED** [review]
Looks OK.

**Review by romanstrobl — APPROVED** [review]
Looks good now.

**Pull request merged**